.pipelines/templates/.builder-release-template.yaml

```diff
@@ -1,13 +1,55 @@
 parameters:
-    - name: artifactName
-      type: string
-      default: 1604
-
+  - name: artifactName
+    type: string
+    default: 1604
+  - name: useOverrides
+    type: boolean
+    default: false
+  - name: overrideBranch
+    type: string
+    default: master
 steps:
+  - checkout: self
```

Comment:
does this incur any extra work in the non-overrides case?

Reply:
Nope. The catch with multiple repositories is once you specify a checkout step, the default checkout step is removed, so this step brings it back. If anything, it might be more performant than before, because we're specifying not to retrieve git history or tags.